src/internal/observable/dom/AjaxObservable.ts

```diff
@@ -411,6 +411,13 @@ export class AjaxSubscriber<T> extends Subscriber<Event> {
       }
     }
     xhr.onload = xhrLoad;
+    xhr.onabort = () => {
+      const { subscriber, progressSubscriber } = (<any>xhrLoad);
+      if (progressSubscriber) {
+        progressSubscriber.complete();
```

Comment:
`complete` is a signal of successful completion. This isn't correct. If it's aborted, it did not complete successfully.